File: src/main/java/com/aliiensmp/aliienResize/Config/Records/CachedSizeItem.java
package com.aliiensmp.aliienResize.Config.Records;

import org.bukkit.inventory.ItemStack;

public record CachedSizeItem(
        int slot,
        String id,
        String permission,
        double scale,
        ItemStack availableItem,
        ItemStack selectedItem,
        ItemStack noPermItem
) {}

File: src/main/kotlin/com/aliiensmp/aliienResize/config/data/CachedSizeItem.kt
package com.aliiensmp.aliienResize.config.data

import org.bukkit.inventory.ItemStack

data class CachedSizeItem(
    val slot: Int,
    val id: String,
    val permission: String,
    val scale: Double,
    val availableItem: ItemStack,
    val selectedItem: ItemStack,
    val noPermItem: ItemStack
)
